executable/odex.ksy

```diff
@@ -0,0 +1,54 @@
+meta:
+  id: odex
+  title: Odex
+  file-extension: odex
+  license: Apache-2.0
+  ks-version: 0.9
+  imports:
+    - /executable/dex
+  endian: le
+  encoding: UTF-8
+doc-ref: http://web.archive.org/web/20180816094438/https://android.googlesource.com/platform/dalvik.git/+/master/libdex/DexFile.h
+seq:
+  - id: magic
+    contents: "dey\n"
+  - id: version
+    type: strz
+    size: 4
+  - id: ofs_dex
+    type: u4
+    doc: file offset of DEX header
+  - id: len_dex
+    type: u4
+  - id: ofs_deps
+    type: u4
+    doc: offset of optimized DEX dependency table
+  - id: len_deps
+    type: u4
+  - id: ofs_opt
+    type: u4
+    doc: file offset of optimized data tables
+  - id: len_opt
+    type: u4
+  - id: flags
```

Comment:
Should it be a type of bit fields?

Reply:
I could not find any information in the header file about what this should be:

http://web.archive.org/web/20180816094438/https://android.googlesource.com/platform/dalvik.git/+/master/libdex/DexFile.h#554